app/views/users/show.html.erb

```diff
@@ -1,9 +1,30 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
+  <% unless @user == current_user %>
+    <span class="user-link">
+      <% if sent_requests(@user.id) %>
+        pending
+      <% else %>
+        <%= link_to 'invite to friendship', user_invite_path(@user), class: 'invite-btn', method: :post%>
+      <% end %>
+    </span>
+  <% end %>
+
   <article class="timeline">
     <h3>Recent posts:</h3>
     <ul class="posts">
       <%= render @posts %>
```

Comment:
Thanks for the review.
Just like you observed, we have implemented the part where the timeline includes posts of the signed in user and all their friends. However, we didn't include that for the show page of a user because the show page of a user should only display posts made by him/her and so far that is how our app works.